### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by tjch-o**

### CODE REVIEW

File: docs/diagrams/HomeSequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+HomeCommand -> Model : getUnfilteredContactList()
+Model --> HomeCommand
+
+HomeCommand -> Model: updateFilteredTaskList()
+Model --> HomeCommand
+
+deactivate Model
+
+HomeCommand --> JobFestGoParser
```
Comment (rionshocker) [inline docs/diagrams/HomeSequenceDiagram.puml:46]: `HomeCommand` should go back to `LogicManager` and not `JobFestGoParser`

File: docs/diagrams/HomeSequenceDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+deactivate HomeCommand
```
Comment (rionshocker) [inline docs/diagrams/HomeSequenceDiagram.puml:47]: Remember to destroy `HomeCommand` as well.

File: docs/diagrams/HomeSequenceDiagram.puml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+HomeCommand -> Model : getUnfilteredEventList()
+activate Model
+
+Model --> HomeCommand
+
+HomeCommand -> Model : getUnfilteredContactList()
+Model --> HomeCommand
+
+HomeCommand -> Model: updateFilteredTaskList()
+Model --> HomeCommand
```
Comment (rionshocker) [inline docs/diagrams/HomeSequenceDiagram.puml:42]: Can consider showing it creating a `CommandResult`

**Review by rionshocker — CHANGES_REQUESTED** [review]
LGTM! Just some nits to be fixed.

File: docs/diagrams/HomeSequenceDiagram.puml (hunk 1 above)
Comment (tjch-o) [reply docs/diagrams/HomeSequenceDiagram.puml]: nice catch thx

File: docs/diagrams/HomeSequenceDiagram.puml (hunk 2 above)
Comment (tjch-o) [reply docs/diagrams/HomeSequenceDiagram.puml:47]: nice catch thx

File: docs/diagrams/HomeSequenceDiagram.puml (hunk 3 above)
Comment (tjch-o) [reply docs/diagrams/HomeSequenceDiagram.puml:43]: oh good suggestion thank you for this!

**Review by rionshocker — APPROVED** [review]
LGTM!

**Pull request merged**